README.md

```diff
@@ -5,164 +5,134 @@
+| [machine_type] | Valid machine type such as "mz2o-2x16", "bz2-4x16", "bz2-8x32", etc... | `string` | "mz2o-2x16" | yes | quickstart, standard |
+| [prefix] | A unique identifier for resources | `string` | n/a | yes | quickstart, standard |
+| [region] | Region where VPC will be created | `string` | n/a | yes | quickstart, standard |
+| [ssh_public_key] | A public SSH Key for VSI creation which does not already exist in the deployment region | `string` | unique | yes | quickstart, standard |
+| [ports] | Inbound port for telnet & zosmf web browser for Wazi VSI SG  | `list(number)` | n/a | yes | quickstart, standard |
+| [image_name] | valid image name for Wazi VSI | `string` | "ibm-zos-2-5-s390x-dev-test-wazi-7" | yes | quickstart, standard |
+| [resource_tags] | Optional list of tags to be added to created resources | `string` | n/a | no | quickstart, standard |
+| [sm_service_plan] | Type of service plan to use to provision Secrets Manager if not using an existing one | `string` | "standard" | no | standard |
+| [root_ca_name] | Name of the Root CA to create for a private_cert secret engine. Only used when var.existing_sm_instance_guid is false | `string` | "root-ca" | no | standard |
+| [intermediate_ca_name] | Name of the Intermediate CA to create for a private_cert secret engine. Only used when var.existing_sm_instance_guid is false | `string` | "intermediate-ca" | no | standard |
+| [certificate_template_name] | Name of the Certificate Template to create for a private_cert secret engine. When var.existing_sm_instance_guid is true, then it has to be the existing template name that exists in the private cert engine | `string` | "my-template" | no | standard |
+| [create_policy] | Set to true to create a new access group (using the value of var.access_group_name) with a VPN Client role | `bool` | true | no | standard |
+| [vpn_client_access_group_users] | List of users in the Client to Site VPN Access Group | `list(string)` | [] | no      standard |
+| [access_group_name] | Name of the IAM Access Group to create if var.create_policy is true | `string` | client-to-site-vpn-access-group | no | standard |
+| [root_ca_max_ttl] | Maximum TTL value for the root CA | `string` | "8760h" | yes | standard |
+| [root_ca_common_name] | Fully qualified domain name or host domain name for the certificate to be created | `string` | "cloud.ibm.com" | no | standard |
+| [resource_group] | Name of the resource group to use for this example. If not set, a resource group is created | `string` | null | no | standard |
+| [cert_common_name] | Fully qualified domain name or host domain name for the private certificate to be created | `string` | n/a | no | standard |
+| [override_json_string] | Override default values with custom JSON. Any value other than an empty string will override all other configuration changes | `string` | "" | no | quickstart, standard |
+| [override] | Override default values with custom JSON template | `bool` | true | no | standard |
```

Comment:
Is there any relationship between override_json_string and override?   If override is true, where is the custom JSON template that will be used?  @Pketaki ^^

Add a sentence to explain the relationship and/or difference between these last two variables.

Reply:
@Nikhil22February - I think you missed this one.

Reply:
When **override** value is true so we are updating the string and adding it to **override_json** file.

Reply:
@Nikhil22February  - I don't understand your answer.   If I set override_json_string to a non-null value, I know that means that will be used to override any other values.  However, what then does the override boolean mean??    How does it relate to override_json_string if at all?  If I'm using this and I set a value for override_json_string do I also need to set override to true?  The descriptions for these two parameters need to be clarified so that a user would know how to use them.

Reply:
@Nikhil22February  - ok, then I think we need to add to the description of override variable to indicate that if it is true, then the content of override.json file will be used as input to the terraform.  The description of the variable in the variables.tf file is more complete than the README.  It says: "Override default values with custom JSON template. This uses the file `override.json` to allow users to create a fully customized environment."  

Also, should the default value really be true for this variable?   @Pketaki

Reply:
@kevinkuhner Yes. Default should be true as we will use json file as input first and then modified data will be passed as input to override_json_string

Reply:
@Nikhil22February @Pketaki  - I still think we need an update to the description for the `override` variable.  Make it match the description that is in the variables.tf file.

Reply:
updated description for **override** and arranged alphabetically.